packages/dd-trace/src/config.js

```diff
@@ -189,9 +189,9 @@ function validateNamingVersion (versionString) {
  * @param {string | string[]} input
  */
 function splitJSONPathRules (input) {
-  if (!input) return
+  if (!input || input.toLowerCase() === 'false' || input === '$') return
   if (Array.isArray(input)) return input
-  if (input === 'all') return []
+  if (input === 'all' || input.toLowerCase() === 'true') return []
```

Comment:
Using false or true is undocumented behavior. I don't think we should do that. It is clearly documented how it is possible to enable it or disable it for now.